src/styles/blocks/_variables.scss

```diff
@@ -0,0 +1,3 @@
+$color-blue: #00acdc;
+$color-gray: #616070;
+$color-black: #060b35;
```

Comment:
Add empty line

Reply:
it is

Reply:
i thing that it is problem with github, i pushed ti for 2 times in vs code is ok, but here is no empty line